changelog/README.md

```diff
@@ -4,6 +4,7 @@
 
 ### 4.6.0 (in progress)
 
+- [bug] JAVA-2657: DriverConfigLoader.programmaticBuilder() doesn't work in OSGi container
```

Comment:
We might try to come up with a better name here, suggestion: "Ability to specify the class loader to use for application-specific classpath resources".